lib/Common/DataStructures/LineOffsetCache.h

```diff
@@ -1,269 +0,0 @@
-//-------------------------------------------------------------------------------------------------------
-// Copyright (C) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
-//-------------------------------------------------------------------------------------------------------
-#pragma once
-
-namespace JsUtil
-{
-    template <typename TAllocator>
-    class LineOffsetCache
```

Comment:
>LineOffsetCache [](start = 10, length = 15)

Any changes to this code besides splitting into .h and .cpp under lib/runtime/base and removing the JsUtil namespace?

Reply:
Yes.  Splitting the data from charOffset/byteOffset pair to separate list, so that the byteOffset don't need to be stored if it is ASCII-7 (which charOffset == byteOffset)